transitmatters/t-performance-dash
# add text & link explaining slow time

## Motivation


https://github.com/transitmatters/t-performance-dash/issues/850
## Changes


/system/slowzones:
![image](https://github.com/transitmatters/t-performance-dash/assets/46619169/3264a3dd-0f9a-4312-9612-0b47e96fce7a)

/[line]/slowzones
![image](https://github.com/transitmatters/t-performance-dash/assets/46619169/badb7a26-c005-477d-9fc4-0fce2b253dd3)


## Testing Instructions

Navigate to the system slowzones page and line slowzone pages and ensure that the explainer text is displayed correctly and that the link works
